bundles/org.openhab.binding.pjlinkdevice/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -84,4 +91,31 @@
 		<description>Select the video mute status</description>
 	</channel-type>
 
-</thing:thing-descriptions>
+	<channel-type id="lampHours">
+		<item-type>Number</item-type>
+		<label>Lamp hours</label>
+		<description>How long the lamp has been in use (in hours)</description>
+		<state readOnly="true" />
```

Comment:
Maybe add a pattern including the unit ?

Reply:
done